astropy/coordinates/tests/test_geodetic_representations.py

```diff
@@ -2,16 +2,65 @@
 
 """Test geodetic representations"""
 import pytest
-from numpy.testing import assert_array_equal
 
+from numpy.testing import assert_array_equal
 from astropy import units as u
+from astropy.units import allclose as quantity_allclose
+from astropy.coordinates.geodetic import BaseGeodeticRepresentation
+
 from astropy.coordinates.earth import (
     GRS80GeodeticRepresentation,
     WGS72GeodeticRepresentation,
     WGS84GeodeticRepresentation,
 )
 from astropy.coordinates.representation import CartesianRepresentation
-from astropy.units import allclose as quantity_allclose
+from astropy.units.tests.test_quantity_erfa_ufuncs import vvd
+
+
```

Comment:
Since you are defining new representations, you need to reset the `REPRESENTATION_CLASSES` dict after each test - otherwise they linger (I think this may be the reason that `test-double` failes). Probably easiest to just import `setup_function` and `teardown_function` from `test_representation`.

Reply:
It is not .... apparently pre-commit removes my import ... I guess I'm doing something wrong....

Reply:
No, you do need to ensure that `REPRESENTATION_CLASSES` is set back to its original contents; your custom classes now cause test failures for `table` (I think, didn't look too carefully). I think you need
```
from astropy.coordinates.tests.test_representation import setup_function, teardown_function
````

Reply:
I did it! I didn't realize ruff deleted it ... I've added ```# ruff : noqa: 401``` but it does not force ruff to do what I want...

Reply:
I've edited `pyproject.toml` ... hope this is not a problem...

Reply:
Sorry, I missed this chain - hope my suggested `# noqa: F401` will do the trick...